### PR TITLE
Doc change subprocess dependent 

### PR DESCRIPTION
**Pull request by sneh-wha**
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

File: docs/docs/en/guide/task/dependent.md [hunk 1]
```diff
@@ -7,20 +7,24 @@ Dependent nodes are **dependency check nodes**. For example, process A depends o
 
 ## Create Task
 
-- Click `Project -> Management-Project -> Name-Workflow Definition`, and click the `Create Workflow` button to enter the DAG editing page.
+- Click `Project Management -> Project Name -> Workflow Definition`, and click the `Create Workflow` button to enter the DAG editing page.
```
Comment (SbloodyS) [inline docs/docs/en/guide/task/dependent.md:10]: Please also add chinese's doc.

**Review by SbloodyS — COMMENTED** [review]
to target specific issue from PR, we recommend use keyword fix: #issue_id or close: #issue_id or closes: #issue_id in your PR describe(not title, just desc). It would not only connect issue to PR but also close issue automatically when PR is be closed.

**Review by zhongjiajie — APPROVED** [review]
LGTM

File: docs/docs/en/guide/task/dependent.md (hunk 1 above)
Comment (zhongjiajie) [reply docs/docs/en/guide/task/dependent.md:10]: Hi @SbloodyS , I think  @sneh-wha can not both read and write Chinese docs. So maybe we should jus ignore the Chinese doc and only change the English one

**Pull request merged**